site/_config.yml

```diff
@@ -8,6 +8,7 @@ destination: local-generated
 twitter_url: https://twitter.com/asfbookkeeper
 
 versions:
+- "4.8.0"
```

Comment:
move this down to `[next_version_placeholder]`

I think the script needs to be updated.

Reply:
done, and marked 4.7.2 as "current stable"